modules/apps/foundation/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/OpenIdConnectSessionImpl.java

```diff
@@ -14,20 +14,21 @@
 
 package com.liferay.portal.security.sso.openid.connect.internal;
 
+import java.io.Serializable;
+
+import com.liferay.portal.security.sso.openid.connect.OpenIdConnectSession;
```

Comment:
The impl should remain in the impl package, it should not be in api

Reply:
I don't understand. This impl class is in the impl module, not the api.

Reply:
oops, disregard